docs/specs/restore.yml

```diff
@@ -5,7 +5,7 @@ inputs:
     dependencies:
       docfx-test-dependencies: https://github.com/docascode/docfx-test-dependencies
 restores:
```

Comment:
>restores [](start = 0, length = 8)

we still needs to validate lock files, how about changing E2ETest.cs to be:

```csharp
var restoredFile = Directory.EnumerateFiles(AppData.RestoreDir, file).Single();
```

Reply:
i will add it in next clean PR

---
In reply to: [200888208](https://github.com/dotnet/docfx/pull/3020#discussion_r200888208) [](ancestors = 200888208)